src/app/lib/views/browser/item.js

```diff
@@ -164,7 +158,7 @@
             var realtype = this.model.get('type');
             var itemtype = realtype.replace('bookmarked', '');
             var modelType = itemtype.charAt(0).toUpperCase() + itemtype.slice(1); // 'Movie', 'Show'
-            var provider = App.Providers.get(this.model.get('provider'));
+            var provider = this.model.get('provider');
```

Comment:
Related to the above comment: if the backbone model has to support the entire provider functions (before there was just a string attached), isnt that memory-consuming? I mean, the provider is quite an heavy object to pass attached to a model, isnt it? Anyway, if this is really wanted, then I think there are other places where model.attributes.provider is still using a string (all should be ported, for consistency)